addons/binding/org.openhab.binding.snapcast/src/main/java/org/openhab/binding/snapcast/internal/data/Client.java

```diff
@@ -0,0 +1,650 @@
+                return false;
+            }
+            Volume other = (Volume) obj;
+            if (muted == null) {
+                if (other.muted != null) {
+                    return false;
+                }
+            } else if (!muted.equals(other.muted)) {
+                return false;
+            }
+            if (percent == null) {
+                if (other.percent != null) {
+                    return false;
+                }
+            } else if (!percent.equals(other.percent)) {
+                return false;
+            }
+            return true;
+        }
+
+    }
+
+    /**
+     * The {@link LastSeen} is a data structure for the api communication.
+     * It provides last seen informations.
+     *
+     * @author Steffen Brandemann - Initial contribution
+     */
+    public static class LastSeen {
+
+        @SerializedName("sec")
+        private Long sec;
+
+        @SerializedName("usec")
+        private Long usec;
+
+        public LastSeen() {
+        }
+
+        /**
+         * @return the sec
+         */
+        public Long getSec() {
+            return sec;
+        }
+
+        /**
+         * @param sec the sec to set
+         */
+        public void setSec(Long sec) {
+            this.sec = sec;
+        }
+
+        /**
+         * @return the usec
+         */
+        public Long getUsec() {
+            return usec;
+        }
+
+        /**
+         * @param usec the usec to set
+         */
+        public void setUsec(Long usec) {
+            this.usec = usec;
+        }
+
+        @Override
+        public int hashCode() {
+            final int prime = 31;
+            int result = 1;
+            result = prime * result + ((sec == null) ? 0 : sec.hashCode());
+            result = prime * result + ((usec == null) ? 0 : usec.hashCode());
+            return result;
+        }
+
+        @Override
+        public boolean equals(Object obj) {
+            if (this == obj) {
+                return true;
+            }
+            if (obj == null) {
+                return false;
+            }
+            if (getClass() != obj.getClass()) {
+                return false;
+            }
+            LastSeen other = (LastSeen) obj;
+            if (sec == null) {
+                if (other.sec != null) {
+                    return false;
+                }
+            } else if (!sec.equals(other.sec)) {
+                return false;
+            }
+            if (usec == null) {
+                if (other.usec != null) {
+                    return false;
+                }
+            } else if (!usec.equals(other.usec)) {
+                return false;
+            }
+            return true;
+        }
+
+    }
+
+    /**
+     * The {@link Snapclient} is a data structure for the api communication.
+     * It provides information about the clients software.
+     *
+     * @author Steffen Brandemann - Initial contribution
+     */
+    public static class Snapclient {
```

Comment:
Would it maybe make more sense to have all these classes in their own file, it will make them small and concise.